### DISCUSSION TITLE
Practica de pasteleria vue

### DISCUSSION THREAD

**Discussion by ErickMontesDK** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#575] [Show and tell]
Buenas noches compañeros, les comparto mi proyecto de pagina de pastelería .

Agradezco la critica constructiva. Este proyecto si estuvo particularmente difícil 

Pagina web: https://black-sand-012e9ca10.1.azurestaticapps.net/#/

Repositorio: https://github.com/ErickMontesDK/FrontEnd-Vue-Vuex-Pasteleria

Los archivos de la repo vienen con comentarios por si le puede servir a alguien

**Comment by RodolfoMorquecho** [comment]
Esta demasiado completa y muy estética, sin duda le daré una checada a tu repo porque hay algunas cosas que me fallan.

**Comment by DanyVeneno** [comment]
Bien es bien completa
